### PR TITLE
k8s: add merge-kubeconfigs.sh script

### PR DESCRIPTION
**Pull request by awly**
This script merges multiple kubeconfigs created with `get-kubeconfig.sh`
(or from anywhere else) into a single `merged-kubeconfig`.

Updates https://github.com/gravitational/teleport/issues/5354

### CODE REVIEW

**Review by webvictim — APPROVED** [review]

**Review by klizhentas — CHANGES_REQUESTED** [review]
if we want this in 6.0 can you folks please rebase of `branch/v6`

**Comment by awly** [comment]
I'll just wait for `master` to be updated with latest docs tomorrow and rebase, since this change isn't tied to 6.0.

**Comment by awly** [comment]
Ping CODEOWNERS

File: docs/pages/kubernetes-access.mdx [hunk 1]
```diff
@@ -46,7 +46,7 @@ proxy must have `kube_listen_addr` set and optionally `kube_public_addr` set.
```
Comment (klizhentas) [inline docs/pages/kubernetes-access.mdx:46]: Can you please review this from the docs section? I removed this part from the docs completely

Comment (klizhentas) [reply docs/pages/kubernetes-access.mdx:46]: I meant all changes to kubernetes-access.mdx, new docs don't reference the script, but it is still helpful to have in examples for some folks who may need it

Comment (awly) [reply docs/pages/kubernetes-access.mdx]: Rebased with your changes and added a new Guide page.
PTAL

**Comment by awly** [comment]
PTAL @klizhentas

**Comment by awly** [comment]
Ping @klizhentas or @russjones for approval

**Review by klizhentas — APPROVED** [review]

**Pull request merged**